File: sprint1/kotlin/lesson10code1.kt
fun solve(array: List<Int>): Int {
    var maximalSum = 0
    for (i in 0 until array.size) {
        val currentSum = array.subList(0, i + 1).sum()
        if (maximalSum < currentSum) {
            maximalSum = currentSum
        }
    }
    return maximalSum
}

File: sprint1/python/lesson10code1.py
def solve(array: List[int]) -> int:
    maximal_sum = 0 # Сумма пустого префикса равна 0
    for i in range(len(array)):
        current_sum = sum[array[0: i + 1])
	      if maximal_sum < current_sum:
            maximal_sum = current_sum
    return maximal_sum
